### PR TITLE
FIX: glmax error in new_img_like with Nifti2Images

### PR DESCRIPTION
**Pull request by KamalakerDadi**
As reported in neurostars tracker https://neurostars.org/t/missing-field-error-in-nilearn-with-nifti-2/1365

A fix to overcome the error regarding header attribute "glmax" which is not present in the Nifti2 images.

### CODE REVIEW

**Comment by bthirion** [comment]
The CrileCI failure seems unrelated.
LGTM.

**Comment by KamalakerDadi** [comment]
>The CrileCI failure seems unrelated.

Seems like there are some import error with very recent sphinx 1.7.0. Need to figure it out.

File: nilearn/image/image.py [hunk 1]
```diff
@@ -638,7 +638,9 @@ def new_img_like(ref_niimg, data, affine=None, copy_header=False):
         header = copy.deepcopy(ref_niimg.header)
         header['scl_slope'] = 0.
         header['scl_inter'] = 0.
-        header['glmax'] = 0.
+        # "glmax" is available only for Nifti1Image
+        if not isinstance(ref_niimg, nibabel.nifti2.Nifti2Image):
```
Comment (GaelVaroquaux) [inline nilearn/image/image.py]: Wouldn't it simply work to do "if 'glmax' in header"?

Comment (KamalakerDadi) [reply nilearn/image/image.py]: That's good point. I am changing that.

Comment (KamalakerDadi) [reply nilearn/image/image.py]: Thanks

**Comment by GaelVaroquaux** [comment]
Good job, @KamalakerDadi !

Merging

**Comment by KamalakerDadi** [comment]
Thank you for merging

**Pull request merged**